### PR TITLE
pacman: add German translation

### PR DESCRIPTION
**Pull request by Hoi15A**
- [X] The page (if new), does not already exist in the repository.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages.de/linux/pacman.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+`pacman --sync --refresh --sysupgrade`
+
+- Ein neues Paket installieren:
+
+`pacman --sync {{paket_name}}`
+
+- Ein Paket und dessen Abhängigkeiten entfernen:
+
+`pacman --remove --recursive {{paket_name}}`
```
Comment (navarroaxel) [inline pages.de/linux/pacman.md:17]:
```suggestion
`pacman --remove --recursive {{paketname}}`
```

File: pages.de/linux/pacman.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+> Manche Unterbefehle wie `pacman sync` sind separat dokumentiert.
+> Weitere Informationen: <https://man.archlinux.org/man/pacman.8.de>.
+
+- Synchronisieren und alle Pakete aktualisieren:
+
+`pacman --sync --refresh --sysupgrade`
+
+- Ein neues Paket installieren:
+
+`pacman --sync {{paket_name}}`
```
Comment (navarroaxel) [inline pages.de/linux/pacman.md:13]: Now, I understand `Paketname` is a single word.
```suggestion
`pacman --sync {{paketname}}`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

File: pages.de/linux/pacman.md (hunk 2 above)
Comment (Hoi15A) [reply pages.de/linux/pacman.md]: also noticed that suchmuster is also a single word so I've fixed that in addition to this.

File: pages.de/linux/pacman.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+# pacman
+
+> Arch Linux Paket Management Tool.
+> Manche Unterbefehle wie `pacman sync` sind separat dokumentiert.
+> Weitere Informationen: <https://man.archlinux.org/man/pacman.8.de>.
+
+- Synchronisieren und alle Pakete aktualisieren:
```
Comment (marchersimon) [inline pages.de/linux/pacman.md:7]: All descripptions should be in imperative form, like this:
```suggestion
- Synchronisiere und aktualisiere alle Pakete:
```

**Review by navarroaxel — APPROVED** [review]

**Review by marchersimon — APPROVED** [review]
Thanks @Hoi15A

**Pull request merged**